layouts/shortcodes/docs/languages/exporters/intro.md

```diff
@@ -1,7 +1,8 @@
-To visualize and analyze telemetry, export it to the
-[OpenTelemetry Collector](/docs/collector/), to a backend such as
-[Jaeger](https://jaegertracing.io/), [Zipkin](https://zipkin.io/),
-[Prometheus](https://prometheus.io/), or a
+To ensure that your telemetry is exporting correctly, send it to the
+[OpenTelemetry Collector](/docs/collector/). It is also considered a best
+practice to use the Collector in production. To visualize your telemetry, export
+it to a backend such as [Jaeger](https://jaegertracing.io/),
+[Zipkin](https://zipkin.io/), [Prometheus](https://prometheus.io/), or a
```

Comment:
```suggestion
Send telemetry to the [OpenTelemetry Collector](/docs/collector/) to
make sure it's exported correctly. Using the Collector in production
environments is a best practice. To visualize your telemetry, export
it to a backend such as [Jaeger](https://jaegertracing.io/),
[Zipkin](https://zipkin.io/), [Prometheus](https://prometheus.io/), or a
```

Reply:
Thank you @theletterf, that's much better! 🙏 
(I wasn't pleased with the text, but didn't have the inspiration to improve it.)